ospf6d/ospf6_top.c

```diff
@@ -678,6 +757,9 @@ DEFUN (ospf6_interface_area,
 	struct ospf6_area *oa;
 	struct ospf6_interface *oi;
 	struct interface *ifp;
+	struct ospf6 *ospf6;
+
+	ospf6 = ospf6_lookup_by_vrf_name(VRF_DEFAULT_NAME);
```

Comment:
Get the context from vty so that this can work for VRF as well.

Reply:
Resolved